src/connection.ts

```diff
@@ -250,6 +259,46 @@ export class Connection extends EventEmitter {
     return stream
   }
 
+  /**
+   * Connections minimum exchange rate with slippage included, if not set '0' is returned.
+   */
+  get minimumAcceptableExchangeRate (): string {
+    if (this.exchangeRate) {
+      const minimumExchangeWithSlippage = this.exchangeRate
+         .times(new BigNumber(1).minus(this.slippage))
+      return minimumExchangeWithSlippage.toString()
+    }
+    return '0'
+  }
+  
+ /**
+  * Calculates the last exchange rate based on last packet successfully sent.
+  */
+  get lastPacketExchangeRate (): string {
+    return this._lastPacketExchangeRate.toString()
+  }
+
+  /**
+   * Total delivered so far, denominated in the connection plugin's units.
+   */
+  get totalDelivered (): string {
+    return this._totalDelivered.toString()
+  }
+
+  /**
+   * Total sent so far, denominated in the connection plugin's units.
+   */
+  get totalSent (): string {
+    return this._totalSent.toString()
+  }
+
+  /**
+   * Total received so far, denominated in the connection plugin's units.
```

Comment:
Might be worth emphasizing that this is how much the local side has received from the remote (delivered and received sound somewhat similar to me)

Reply:
Good point, updated it to mention the received is for the local side.